sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/tests/Tests/ContainerRegistryTests.cs

```diff
@@ -809,5 +816,213 @@ void ValidateSystemData(SystemData systemData)
+                    resourceGroup.Name,
+                    registry.Name,
+                    tokenName,
+                    new Token(scopeMapId: scopeMap.Id)
+                );
+
+                // Create connected registry
+                var connectedRegistry = ContainerRegistryTestUtilities.CreatedContainerRegistryConnectedRegistry(registryClient, resourceGroup.Name, registry.Name, connectedRegistryName, token.Id);
+                Assert.NotNull(connectedRegistry);
+                Assert.NotEmpty(connectedRegistry.Id);
+                Assert.Equal("Inactive", connectedRegistry.Activation.Status);
+                Assert.Equal("Offline", connectedRegistry.ConnectionState);
+                Assert.Equal("Disabled", connectedRegistry.Logging.AuditLogStatus);
+                Assert.Equal("Information", connectedRegistry.Logging.LogLevel);
+                Assert.Equal("ReadWrite", connectedRegistry.Mode);
+                Assert.Equal(ProvisioningState.Succeeded, connectedRegistry.ProvisioningState);
+                Assert.Equal(token.Id, connectedRegistry.Parent.SyncProperties.TokenId);
+                Assert.Equal("0 9 * * *", connectedRegistry.Parent.SyncProperties.Schedule);
+                Assert.Equal(XmlConvert.ToTimeSpan("PT48H"), connectedRegistry.Parent.SyncProperties.MessageTtl);
+                Assert.Equal(XmlConvert.ToTimeSpan("PT4H"), connectedRegistry.Parent.SyncProperties.SyncWindow);
+
+                // List connected registries
+                var connectedRegistries = registryClient.ConnectedRegistries.List(resourceGroup.Name, registry.Name);
+                connectedRegistry = connectedRegistries.First(cr => StringComparer.OrdinalIgnoreCase.Equals(cr.Name, connectedRegistry.Name));
+                Assert.Single(connectedRegistries);
+                Assert.NotNull(connectedRegistry);
+
+                // Get connected registry
+                connectedRegistry = registryClient.ConnectedRegistries.Get(resourceGroup.Name, registry.Name, connectedRegistry.Name);
+                Assert.NotNull(connectedRegistry);
+
+                // Update connected registry
+                connectedRegistry = registryClient.ConnectedRegistries.Update(resourceGroup.Name, registry.Name, connectedRegistry.Name, new ConnectedRegistryUpdateParameters(
+                    syncProperties: new SyncUpdateProperties(
+                        messageTtl: XmlConvert.ToTimeSpan("PT24H"),
+                        syncWindow: XmlConvert.ToTimeSpan("PT2H"))
+                    )
+                );
+                Assert.Equal(XmlConvert.ToTimeSpan("PT24H"), connectedRegistry.Parent.SyncProperties.MessageTtl);
+                Assert.Equal(XmlConvert.ToTimeSpan("PT2H"), connectedRegistry.Parent.SyncProperties.SyncWindow);
+
+                // Delete connected registry
+                registryClient.ConnectedRegistries.Delete(resourceGroup.Name, registry.Name, connectedRegistry.Name);
+
```

Comment:
@AntonioVT please delete the scopeMap and token resources here as well.

Reply:
Sure! I will delete resources dependencies and then resource. 
ie: (Connected registry test: Delete scope map, token and then connected registry. Token test: Delete scope map and then token)

Reply:
@AntonioVT for this case, the connected registry must be deleted first. For token resources, RP blocks delete if there exists a dependent connected registry that uses that resource as a sync or client token.

Reply:
Token test had a dependency on a built-in scope map, so can't delete that one. For the connected registry test, deleted token and scope map. Resolving conversation, thanks!